monai/bundle/workflows.py

```diff
@@ -285,6 +286,10 @@ def run(self) -> Any:
         Run the bundle workflow, it can be a training, evaluation or inference.
 
         """
+        _bundle_root_path = (
+            self.config_root_path.parent if self.config_root_path.name == "configs" else self.config_root_path
+        )
+        sys.path.append(str(_bundle_root_path))
```

Comment:
we may consider `sys.path.insert(0, path)` to make the newly added path higher priority. please also update the docstring about this sys path change in this function.

Reply:
When setting PYTHONPATH these values get placed in `sys.path` after an initial empty string but before everything else, so perhaps `sys.path.insert(1, path)` would make more sense. Unless names clash anywhere it wouldn't actually matter. let's leave this for now since users can always set PYTHONPATH to prepend things, but yes we should also document th behaviour in the docstring.